### ISSUE TITLE
Project global defaults

### ISSUE THREAD

**Issue by jonscottfilm** [karmaradio/karma#93]
Ability for a project admin to set defaults per project: 

The more control we give the project user, the easier creating offers (i.e. not repetitive data input) will be for them, and the easier ALL contractor users will be connected in this way.

1. Working Week - 5 Days (default, but can change per offer)
2. Currency - GBP (default, but can change per offer)
3. Sixth Day Multiplier
4. Seventh Day Multiplier

There will be more, e.g: 

- Per Diem, per country of found locations (cost to eat varies from New York to Torquay)
- Shooting overtime rate on continuous day
- Shooting overtime rate on 12hr day (11hr shoot, 1hr lunch)

**Comment by jackcarlisle** [comment]
@jonscottfilm I agree that this could be a beneficial feature to have. However I feel like it's a "nice to have" compared to an "essential" at this stage. We can definitely add it to the backlog because it's something that might need to be verified by user testing as well.

**Comment by jonscottfilm** [comment]
Absolutely.
